deployer/src/handlers/mod.rs

```diff
@@ -621,6 +623,19 @@ pub async fn start_deployment(
         };
         deployment_manager.run_push(built).await;
 
+        tokio::spawn(async move {
+            let event =
+                async_posthog::Event::new("shuttle_api_start_deployment".to_string(), account_name);
```

Comment:
```suggestion
                async_posthog::Event::new("shuttle.start_deployment".to_string(), account_name);
```
I'm not sure what the PostHog conventions are for naming events. So just a suggestion.

Reply:
This actually resembles the naming convention we already use. For example:
`console_project_back`